src/dpq2/oids.d

```diff
@@ -140,16 +144,24 @@ bool isSupportedArray(OidType t) pure
 
 OidType detectOidTypeFromNative(T)()
 {
+    import std.datetime.date : StdDate = Date, TimeOfDay;
+    import std.datetime.systime : SysTime;
+    import std.traits : Unqual;
+
     with(OidType)
     {
-        static if(is(T == string)){ return Text; } else
-        static if(is(T == ubyte[])){ return ByteArray; } else
-        static if(is(T == bool)){ return Bool; } else
-        static if(is(T == short)){ return Int2; } else
-        static if(is(T == int)){ return Int4; } else
-        static if(is(T == long)){ return Int8; } else
-        static if(is(T == float)){ return Float4; } else
-        static if(is(T == double)){ return Float8; } else
+        static if(is(Unqual!T == string)){ return Text; } else
```

Comment:
As for me, too many "Unqual" copy'n'pasts, can we replase it to something as:
```d
OidType detectOidTypeFromNative(NativeT)()
[...]
alias T = Unqual!NativeT;
```
?

Reply:
Agreed